### PR TITLE
Update carthage readme

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

File: Demo/EarlGreyExample/README.md [hunk 1]
```diff
@@ -21,9 +21,17 @@ Run the tests that you want to see in action.
...
 - `open EarlGreyExample.xcodeproj`
 
-Xcode should now allow you to execute the EarlGreyExampleSwiftTests tests.
+Xcode should now allow you to execute the EarlGreyExampleSwiftTests tests. If you get a compile error, try the following:
+
+- Delete derived data
+  - `rm -rf "$HOME/Library/Developer/Xcode/DerivedData"`
+- Delete untracked and modified data
+  - `git reset --hard; git clean -dfx`
+- Restart the computer
```
Comment (khandpur) [inline Demo/EarlGreyExample/README.md]: seriously?

Comment (bootstraponline) [inline Demo/EarlGreyExample/README.md]: seriously, ask anyone who has used Xcode for years 😄

Comment (bootstraponline) [inline Demo/EarlGreyExample/README.md]: or Apple's official release notes:


`Restart your computer` is a frequent recommendation for issues.
https://developer.apple.com/library/content/documentation/Xcode/Conceptual/RN-Xcode-Archive/Chapters/xc4_release_notes.html

**Comment by bootstraponline** [comment]
ok I removed it for now.

**Pull request merged**